### DISCUSSION TITLE
custome week link  format like 2023/07/W27 instead of default(2023-W27)

### DISCUSSION THREAD

**Discussion by polandeme** [YU000jp/logseq-plugin-show-weekday-and-week-number#64] [Announcements]
I already have a similar page for week page, like  2023/07/W27. 
can id custome week link by myself.

<img width="654" alt="image" src="https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/assets/4548308/d35ac7f2-00da-44d1-8cb5-05296823b1c2">

**Comment by YU000jp** [comment]
The current week numbering format we are using is based on ISO8601. I think that adopting the format you are proposing would enhance convenience, as it would enable the use of the Hierarchy feature in Logseq. We will consider incorporating it as an option.

**Reply by YU000jp** [reply]
To achieve this, a new logic for week numbering is required due to several issues. The problems are as follows:

1. Handling weeks that span across months (assigning them to the previous month).
2. Handling weeks that span across years (in the ISO format, the week containing January 4th is considered the first week of the year, while other days are included in the last week of the previous year).

Considering the second issue, it is anticipated that using the hierarchy would involve a more complex format like "2023/07/2023-W27." Alternatively, it might be worth considering a different approach using alias or tags in page property.

**Reply by YU000jp** [reply]
DONE https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/issues/65 (Add the month links to page-tags property)

**Reply by polandeme** [reply]
Sorry, my ability to write in English is fairly average, but I can understand your replies.

I failed to consider these two issues. (-_-
Whether can provide users with an option about these two problems：
Week number calculate by  first day or last day of week,  because this issue #65 also have above question about week number.

**Reply by hodanli** [reply]
I understand your concerns. For weeks that span across months, we could use the first day of the week to determine its placement. For example, if a week has its first three days in October and the last four in November, we can categorize it under October. The same logic can apply for years. While this might not be perfect, it significantly improves usability. I suggest adding this feature as an option.

**Reply by YU000jp** [reply]
@hodanli
Sometimes only one day belongs to the previous month, which feels strange. I understand that you want to sort by month.
